src_features/performPrivacyOperation/ui_common_performPrivacyOperation.c

```diff
@@ -0,0 +1,27 @@
+#include "shared_context.h"
+#include "feature_getPublicKey.h"
+#include "ui_callbacks.h"
+
+unsigned int io_seproxyhal_touch_privacy_ok(__attribute__((unused)) const bagl_element_t *e) {
```

Comment:
why put a used attribute and not directly delete it?

Reply:
the function definition follows bagl callbacks convention (lib_ux/include/ux.h in the SDK) but the callback parameter isn't used here